src/commands/install.yml

```
@@ -1,22 +1,26 @@
 description: >
-  Install MATLAB on a Linux machine executor. Currently, this command is available only for public
-  projects and does not include transformation products, such as MATLAB Coder and MATLAB Compiler.
+  Install a specified version of MATLAB. If you use this command to install transformation products, such
+  as MATLAB Coder and MATLAB Compiler, the command does not automatically license such products for you.
 
 parameters:
   products:
     description: >
-      List of MathWorks products to install.
+      Products to install in addition to MATLAB, specified as a list of product names separated by
+      spaces. You can specify products to install most MathWorks products and support packages. The
```

Without it being clear that "products" refers to the param name here, the sentence reads strangely. If we can't use backtick, perhaps quotes or "You can specify this parameter ..." would be better.

```suggestion
      spaces. You can specify `products` to install most MathWorks products and support packages. The
```

---

Updated to "this parameter"